chroma_core/lib/service_config.py

```diff
@@ -317,7 +317,43 @@ def _setup_influxdb(self):
         log.info("Creating InfluxDB database...")
         self.try_shell(["influx", "-execute", "CREATE DATABASE {}".format(settings.INFLUXDB_IML_DB)])
         self.try_shell(["influx", "-execute", "CREATE DATABASE {}".format(settings.INFLUXDB_STRATAGEM_SCAN_DB)])
+        self.try_shell(
```

Comment:
Given we only store the most recent scan result for stratagem, I wonder if we need to have a retention policy for it.

Reply:
What would be the downside of using the 90 day retention policy?

Reply:
Stats get dropped after 90 days. Though they would probably be stale by that point anyway.